### PR TITLE
Peters changes

### PR DESCRIPTION
**Pull request by pchocho**
Breaking out setup instructions into separate pages and adding them to the side bar nav. Added an 'overview' page at top. Also multiple copy edits. Added Twitter and copyright to footer.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,443 +1,20 @@
...
-Unfortunately, if third-party cookies are blocked, our authentication process does not work.
-
-If you have disabled cross-site cookies, you'll need to add the following to the "allow these sites to always use cookies" list in Chrome settings to get CodeSee to work:
-
-```
-app.codesee.io
-[*.]okta.com
-```
-
+- If it delays for long enough, your pop-up blocker may decide to block it. Keep an eye out for that if the codesee site isn't coming up. If in doubt, you can always go to [app.codesee.io/library](https://app.codesee.io/library) to see all your latest recordings.
```
Comment (jleven) [inline docs/index.md:19]: This was meant to be a sub-bullet... but, if a top-level bullet, maybe just add another word or two for context:

"If the buffering takes long enough"

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- <a href="/setup-cra" target="_blank">**Create React App**</a>
+- <a href="/setup-typescript-without-babel" target="_blank">**Typescript, but without Babel**</a>
+- <a href="/setup-nuxt" target="_blank">**Nuxt**</a>
+- <a href="/setup-storybook" target="_blank">**Storybook**</a>
+- <a href="/setup-gatsby" target="_blank">**Gatsby**</a>
+- <a href="/setup-ember" target="_blank">**Ember**</a>
+
+Or, select generic setup if you use Next.js, Typescript with Babel, or another framework not listed above:
+
+- <a href="/setup-generic" target="_blank">**Generic setup**</a>
```
Comment (jleven) [inline docs/installation.md:39]: Do we know these links will work, or will we need to try that out?

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+## Step 3: Rebuild and run your app locally
+Rebuild your app, wait a few seconds, and you should see the CodeSee eye icon towards the top right of your screen. Then you can celebrate!
+
+A few thoughts to help you on your CodeSee journey:
+
+- If the CodeSee icon is in a red circle then you're not logged in. Click it to log into CodeSee.
+- If the CodeSee icon is in a purple circle, you're ready to go! Click it to start recording, interact with your app, then click it again to stop recording.
+- Keep recordings as short as possible -- focus on the one thing you want to understand better.
+- When you finish recording, CodeSee will open a new tab with your recording in it as soon as all the data has been transmitted. For larger recordings or on slower networks, there can be some buffering, so please give it a few seconds or more to finish sending the recording to our servers.
+- If it delays for long enough, your pop-up blocker may decide to block it. Keep an eye out for that if the codesee site isn't coming up. If in doubt, you can always go to [app.codesee.io/library](app.codesee.io/library) to see all your latest recordings.
```
Comment (jleven) [inline docs/installation.md:50]: ditto the "add context" comment from before

File: docs/setup-cra.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+-   "start": "react-scripts start",
++   "start": "react-app-rewired start",
+-   "build": "react-scripts build",
++   "build": "react-app-rewired build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+}
+```
+
+Note: Please avoid setting this call for the `eject` and `test` scripts.
```
Comment (jleven) [inline docs/setup-cra.md:84]: I worry a little that users will get to the end of this and not be able to find step 3 - rebuild your app. Any ideas how we can help direct them there? Maybe a "Next step -- Rebuild your app" link?

**Review by jleven — APPROVED** [review]
This is awesome, thank you!!

I left a couple comments, but nothing blocking. This is strictly better than what we have today. Just lovely

File: docs/index.md (hunk 1 above)
Comment (pchocho) [reply docs/index.md:19]: I saw it was indented more in the markdown, but it wasn't being displayed as a sub-bullet. I can look into it.

Comment (pchocho) [reply docs/index.md:19]: Ah it needs 4 spaces. Fixing

File: docs/installation.md (hunk 2 above)
Comment (pchocho) [reply docs/installation.md:39]: Adding the step back to this page at the end of each of these sub pages, so will change these to markdown links that open in the same window.

File: docs/setup-cra.md (hunk 4 above)
Comment (pchocho) [reply docs/setup-cra.md:84]: Good point, adding a link at the end back to the installation page, step 3

**Pull request merged**